src/main/java/io/cryostat/net/web/http/api/beta/RecordingsFromIdPostHandler.java

```diff
@@ -170,6 +178,11 @@ public void handleAuthenticated(RoutingContext ctx) throws Exception {
             throw new ApiException(503, "Recording saving not available");
         }
 
+        int maxFiles = Integer.parseInt(ctx.request().getParam("maxFiles", globalMaxFiles));
```

Comment:
We need to catch exception where the `maxFiles` are not a number? like `maxFiles=abc`?